src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -411,35 +401,38 @@ void DoExecute ()
 				? CachePath
 				: Path.Combine (Environment.GetFolderPath (Environment.SpecialFolder.LocalApplicationData), CacheBaseDir);
 
-				using (var resolver = new DirectoryAssemblyResolver (this.CreateTaskLogger (), loadDebugSymbols: false)) {
-					foreach (var assemblyItem in Assemblies) {
-						string fullPath = Path.GetFullPath (assemblyItem.ItemSpec);
-						if (DesignTimeBuild && !File.Exists (fullPath)) {
-							LogWarning ("Failed to load '{0}'. Check the file exists or the project has been built.", fullPath);
-							continue;
-						}
-						if (assemblies.Contains (fullPath)) {
-							LogDebugMessage ("  Skip assembly: {0}, it was already processed", fullPath);
-							continue;
-						}
-						// don't try to even load mscorlib it will fail.
-						if (string.Compare (Path.GetFileNameWithoutExtension (fullPath), "mscorlib", StringComparison.OrdinalIgnoreCase) == 0)
-							continue;
-						assemblies.Add (fullPath);
-						resolver.Load (fullPath);
+				foreach (var assemblyItem in Assemblies) {
+					string fullPath = Path.GetFullPath (assemblyItem.ItemSpec);
+					if (DesignTimeBuild && !File.Exists (fullPath)) {
+						LogWarning ("Failed to load '{0}'. Check the file exists or the project has been built.", fullPath);
```

Comment:
I think we need to make this a LogCodedWarning if possible

Reply:
Indent some code, then you have to write docs! 😜

Reply:
Hmm. I think for this exact `LogWarning()` in `DoExecute()` I temporarily skipped further work on it when I realized that the warnings in other tasks might be a higher priority because the Xamarin-authored NuGet packages have been using the Xamarin.Build.Download mechanism for a while instead of the `GetAdditionalResourcesFromAssemblies` mechanism. I was going to circle back eventually to the remaining uses of `LogWarning()` in `GetAdditionalResourcesFromAssemblies`.

Reply:
Ok let me go ahead and add the warning codes for these then, thanks.